### PR TITLE
added form validation and tweaked content of pages

### PR DESCRIPTION
**Pull request by malawi-mike**
added form validation to one_time_payment_view.php but need your help connecting the donate button to stripe. once you fix that, I'll add form validation to the village fund payment view

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -7,10 +7,11 @@
...
 <meta property="fb:appid" content="<?php print FACEBOOK_APP_ID; ?>"/>
 <meta property="og:type" content="website" />
-<meta property="og:image" content="https://4and.me/images/header1.jpg"/>
-<meta property="og:title" content="Village X"/>
+<meta property="og:image" content="temp/IMG_1707.jpg"/>
+<meta property="og:title" content="This website distrupts extreme poverty in Africa."/>
 <meta property="og:url" content="https://4and.me/"/>
-<meta property="og:description" content="Fund Projects That Villages Choose - because everyone deserves a voice in development"/>
+<meta property="og:description" content="Village X Org is like GoFundMe for rural Africa. Villages choose and partially finance the projects."/>
+
```
Comment (jdepree) [inline index.php]: Since these are now in header.inc, we should delete them from here.

Comment (malawi-mike) [reply index.php]: cool. want me to do it?

**Comment by jdepree** [comment]
yup

**Comment by jdepree** [comment]
You can add commits to an existing branch - that will be a lot cleaner than creating a new branch each time.

**Pull request closed without merging**